src/lib/viz/source/SQLSource.ts

```diff
@@ -214,6 +215,10 @@ export class SQLSource extends Source {
     }
   }
 
+  public isEmpty() {
```

Comment:
Nice, I think this is useful. Although I see the default method in the generic Source, I'm missing it in the GeoJSON one. I think we should try to keep the Sources as aligned as possible. What do you think? In that case I would return something like "features.length === 0"…